fleetctl/fleetctl.go

```diff
@@ -845,3 +845,28 @@ func suToGlobal(su schema.Unit) bool {
 	}
 	return u.IsGlobal()
 }
+
+// checkUnitNames sanity checks unit names
+func checkUnitNames(names []string, allowTemplate bool) error {
+	units := make([]string, 0, len(names))
```

Comment:
yeah, might be best to write some tests that demonstrate which names fail the check and which succeed